Tests/conf.json

```diff
@@ -1474,6 +1474,10 @@
             "memory_threshold": 200,
             "instance_names": "use_python_requests_handler"
         },
+        {
+            "integrations": "ConcentricAI",
+            "playbookID": "ConcentricAI Demo Playbook"
+        },
```

Comment:
No need, as we do not run this test in our nightly.
```suggestion
```

Reply:
@bakatzir could you please elaborate ?

Reply:
Indeed. you can add to the integration yml:
```
tests:
- No tests
```
We are working to fix it in the next demisto-sdk release, as it is not required of partners.

Reply:
Done.